config-service-impl/src/main/java/org/hypertrace/config/service/store/DocumentConfigStore.java

```diff
@@ -158,6 +158,14 @@ public List<UpsertedConfig> writeAllConfigs(
     return this.writeConfigs(resourceContextValueMap, userId);
   }
 
+  @Override
+  public void deleteConfigs(Set<ConfigResourceContext> configResourceContexts) {
+    if (configResourceContexts.isEmpty()) {
+      return;
+    }
+    collection.delete(getConfigResourceContextsFilter(configResourceContexts));
```

Comment:
Should we also return and maybe log the total deleted documents returned by this doc store method? 🤔

Reply:
we are returning the documents before after get. Also delete doesnt return the count. it only returns the boolean.

Reply:
Ah, got it. I think bulk delete returns `BulkDeleteResult` which has the deleted count.